### ISSUE TITLE
Error when calling static method with namespace qualification

### ISSUE THREAD

**Issue by Bretttt** [koenbeuk/EntityFrameworkCore.Projectables#85]
I created a static method

```
public static class Functions
{
    [Projectable]
    public static int Foo(int v1, int v2) => v1 + v2;
}

```

and an entity

```
public class MyEntity
{
    public int Prop1 { get; set; }
    public int Prop2 { get; set; }

    [Projectable]
    public int Projection => Functions.Foo(Prop1, Prop2);
}

```
Everything works fine.

However, when I qualify the method call using the namespace,

```
public class MyEntity
{
    public int Prop1 { get; set; }
    public int Prop2 { get; set; }

    [Projectable]
    public int Projection => My.Namespace.Functions.Foo(Prop1, Prop2);
}
```

It fails.

'Unable to resolve generated expression for MyEntity.Projection'

**Comment by Bretttt** [comment]
I found this also applies to namespace-qualified enums

**Comment by koenbeuk** [comment]
This is fixed with #101
